conda-envs/environment-dev.yml

```diff
@@ -21,7 +21,9 @@ dependencies:
 - typing-extensions>=3.7.4
 # Extra dependencies for dev, testing and docs build
 - ipython>=7.16
-- jax
+# Jaxlib version must not be greater than jax version!
+- jaxlib==0.4.16
+- jax==0.4.16
```

Comment:
Why is jax in the dependencies in the first place?

Reply:
There's a separate job that runs jax tests IIRC

Reply:
There is a separate job, yes, but it installs from the same environment files.

This whole issue is probably due to funny versioning strategies by Google, and incomplete compatibility pins in their feedstocks..